pkg/phlaredb/block/testutil/create_block.go

```diff
@@ -56,3 +64,56 @@ func CreateBlock(t testing.TB, generator func() []*testhelper.ProfileBuilder) (b
 	require.NotNil(t, meta)
 	return *meta, localDir
 }
+
+func OpenBlockFromMemory(t *testing.T, dir string, minTime, maxTime model.Time, profiles, tsdb, symbols []byte) *phlaredb.BlockQuerier {
```

Comment:
We probably should update the tests later to not depend on the phlaredb query implementation

Reply:
TBH, I've already started rewriting segment tests with BlockReader, but it is not ready yet. Will do later.